dask/dataframe/tests/test_rolling.py

```diff
@@ -63,6 +63,11 @@ def basic_rolling_tests(p, d): # Works for series or df
     eq(p.rolling(1).sum(), d.rolling(1).sum())
     # Test with kwargs
     eq(p.rolling(3, min_periods=2).sum(), d.rolling(3, min_periods=2).sum())
+    # Test with center
+    eq(p.rolling(3, center=True).max(), d.rolling(3, center=True).max())
+    eq(p.rolling(3, center=False).std(), d.rolling(3, center=False).std())
+    eq(p.rolling(6, center=True).var(), d.rolling(6, center=True).var())
+    eq(p.rolling(7, center=True).skew(), d.rolling(7, center=True).skew())
```

Comment:
We're strongly considering dropping support for older versions of Pandas (please weigh in here https://github.com/dask/dask/issues/1240 if you have thoughts)